draft-ietf-jsonpath-base.md

```diff
@@ -538,8 +538,9 @@ requirements posed by this document, which are:
 
 1. Integer numbers in the JSONPath query that are relevant
 to the JSONPath processing (e.g., index values and steps) MUST be
-within the range of exact values defined in I-JSON {{-i-json}}, namely
-within the interval \[-(2<sup>53</sup>)+1, (2<sup>53</sup>)-1].
+within the range of exact integer values defined in I-JSON (see {{Section
+2.2 of -i-json}}), namely within the interval \[−(2<sup>53</sup>)+1,
+(2<sup>53</sup>)−1].
```

Comment:
Is "integer" really necessary here?  "Integer numbers" is already the context from line 539.

Reply:
It is not necessary (that's why it wasn't in there), but we have at least one data point that a reader would consider it helpful.  Do you see a downside?

Reply:
Not really. Just felt redundant.